File: assignment 10 - C# practice problems/23.cs
//Write a c# function to get difference between two dates in days.

using System;
namespace MyNameSpace
{
    class Program23
    {
        public static void DifferenceBetweenTwoDatesInDaysDriverFunction()
        {


            Console.WriteLine("Enter 2 dates in dd/mm/yyyy format");
            var dateInput1 = Console.ReadLine();
            var dateInput2 = Console.ReadLine();
            var split1 = dateInput1.Split('/');
            var split2 = dateInput2.Split('/');

            var date1Obj1 = new DateTime(int.Parse(split1[2]), int.Parse(split1[1]), int.Parse(split1[0]), 12, 0, 0);
            var date1Obj2 = new DateTime(int.Parse(split2[2]), int.Parse(split2[1]), int.Parse(split2[0]), 12, 0, 0);


            Console.WriteLine("difference in days is: " + Math.Abs((date1Obj1 - date1Obj2).TotalDays));

        }
    }
}

File: assignment 3 - js practice problems/web-dev-starter/23.js
//Write a JavaScript function to get difference between two dates in days.
function compareTwoDates(dateParam1, dateParam2) {

    let date1 = new Date(dateParam1[2], dateParam1[1], dateParam1[0]);
    let date1Time = date1.getTime();

    let date2 = new Date(dateParam2[2], dateParam2[1], dateParam2[0]);
    let date2Time = date2.getTime();

    let differenceIndateTimes = Math.abs(date1Time - date2Time);

    days = parseInt(differenceIndateTimes / (1000 * 60 * 60 * 24));

    return days;
}


function compareTwoDatesDriverFunction() {
    let dateInput1 = prompt("Enter date1 in dd/mm/yyyy");
    let dateInput2 = prompt("Enter date2 in dd/mm/yyyy");

    let differenceDays = compareTwoDates(dateInput1.split("/"), dateInput2.split("/"));

    alert("The difference in days is " + differenceDays);
}
compareTwoDatesDriverFunction();